.travis.yml

```diff
@@ -3,8 +3,8 @@ sudo: false
 
 env:
     global:
-    - BYOND_MAJOR="510"
-    - BYOND_MINOR="1346"
+    - BYOND_MAJOR="511"
+    - BYOND_MINOR="1379"
```

Comment:
change this to 1380, the version that will have the fix. its now out now, so travis will fail, but this shouldn't get merged until that goes live.